programmers/난이도별/level01.다트_게임/develop-sell.js

```diff
@@ -0,0 +1,101 @@
+function solution(dartResult) {
+  let answer = 0;
+  let temp = 0;
+  let checkoption = true;
+  let post = 0;
+  
+  const pow = (num, upper) => (Math.pow(num, upper))
+  
+  //삼항연산자로 3개의 조건을 걸 수 있는 방법은 없는가? 
+  function pow_num (num, char){
+      if(char === 'S')
+          return num
+      else if (char === 'D')
+          return pow(num, 2)
+      else
+          return pow(num, 3)
```

Comment:
```suggestion
return (char === 'S') ? num
            : ((char === 'D') ?  pow(num, 2) : pow(num, 3));
```
삼항연산자의 중첩은 좋은 컨벤션이 아니라고 하지만 이런식으로도 사용할 수 있을 것 같아요

Reply:
아 삼항연산자 중첩은 좋은 컨벤션이 아니군요.. 알겠습니다!